src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateSplit = date.split(fromFormat[3]);
+  const newDate = [];
+
+
+  for (let i = 0; toFormat.length - 1 > i; i++) {
+    const format = toFormat[i];
+    console.log(format)
+    console.log(newDate)
+
+    if (format === 'YYYY' && fromFormat.includes('YY')) {
+      let year = dateSplit[fromFormat.indexOf('YY')]
+      if (year < 30) {
+        year = '20' + year;
+      } else {
+        year = '19' + year;
+      }
+      newDate.push(year);
+    } else if (format === 'YY') {
+      newDate.push(dateSplit[fromFormat.indexOf('YYYY')].slice(-2));
+    } else {
+      newDate.push(dateSplit[fromFormat.indexOf(format)]);
+    }
+  }
```

Comment:
The code does not handle the case where `fromFormat` contains 'YYYY' but `toFormat` wants 'YY'. This case needs to be added to the conditional logic.